### PR TITLE
Re-post original Code for Hong Kong blog post

### PR DESCRIPTION
**Pull request by debuggingfuture**

### CODE REVIEW

File: _posts/2013-12-01-code-for-hk.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+author: vincentlaucy
+author_url: http://github.com/vincentlaucy
+title: "Code for Hong Kong"
+tagline: "Code Hong Kong better, together"
+tags : [code4hk, blog]
+---
+
+This is a [re-post](http://kleineblase.wordpress.com/2013/12/01/code-for-hong-kong/)
+
+###tl;dr
```
Comment (hupili) [inline _posts/2013-12-01-code-for-hk.md:13]: what's the `tl;dr`?

Comment (debuggingfuture) [inline _posts/2013-12-01-code-for-hk.md:13]: http://www.urbandictionary.com/define.php?term=tl%3Bdr

Comment (hupili) [inline _posts/2013-12-01-code-for-hk.md:13]: I see... didn't know this before. I thought it was some strange coding like HTML entities..

**Comment by hupili** [comment]
@gilbertwat @2blam to try processing PR.

**Pull request merged**